README.md

```diff
@@ -0,0 +1,275 @@
+session is closed and a new one of the correct type is opened and persisted.
+All sessions and clients are closed automatically when the notebook's kernel
+is terminated. Note that opening a schema session to a TypeDB server requires
+that no other sessions be open, regardless of the client used, and once open,
+no other sessions can be opened on the server until the schema session is closed.
+
+## Configuring options
+
+Certain options can be configured using the `%config` magic with:
+
+```
+%config <config argument>`
+```
+
+After being set, these options persist for the remainder of the notebook unless
+changed again. The following table describes the available arguments:
+
+| Argument                                 | Usage                                                                         | Default |
+|------------------------------------------|-------------------------------------------------------------------------------|---------|
+| `TQLMagic`                               | List config options and current set values.                                   |         |
+| `TQLMagic.create_database=<boolean>`     | Create database when opening a connection if it does not already exist.       | `True`  |
+| `TQLMagic.global_inference=<boolean>`    | Enable rule inference for all queries. Can be overridden per query with `-i`. | `False` |
+| `TQLMagic.show_connection=<boolean>`     | Always show current connection name when executing a query.                   | `True`  |
+| `TQLMagic.strict_transactions=<boolean>` | Require session and transaction types to be specified for every transaction.  | `False` |
+
+## Command glossary 
+
+The following tables list the arguments that can be provided to the `%tql` magic command:
+
+| Argument                | Usage                                                                         | Argument type        |
+|-------------------------|-------------------------------------------------------------------------------|----------------------|
+| `-a <server address>`   | TypeDB server address for new connection.                                     | Connection argument. |
+| `-d <database name>`    | Database name for new connection.                                             | Connection argument. |
+| `-n <connection name>`  | Custom name for new connection, or name of existing connection to select.     | Connection argument. |
+| `-l`                    | List currently open connections.                                              | Connection argument. |
+| `-c <connection name>`  | Close a connection by name.                                                   | Connection argument. |
+| `-k <connection name>`  | Close a connection by name and delete its database.                           | Connection argument. |
+| `-p <thread count>`     | Number of server communication threads for new connection (advanced feature). | Connection argument. |
```

Comment:
To be honest I have no idea what the thread count is for. I've marked it as advanced feature to deter users from using it. Does anyone actually make use of this? If not we could just remove it altogether.

Reply:
This only starts to matter for loading "a lot" of data. Someone might potentially try and do this given we've provided an `-f` file path argument. If you see that as a use case for this, then keep it (but try and provide some clear guidance on how to decide on a good value for `-p`.

Reply:
I don't think we have any guidance. The current docs have exactly one line on the topic telling you that it's equal to the number of threads to use for server communication, and the new docs don't have anything on it yet. If you can point me to the relevant guidance then I'll put it in, otherwise I reckon leaving it with `advanced feature` is best. People don't tend to tuch advanced features without specific intent.

Reply:
Removed as discussed.